arangod/IResearch/IResearchFeature.cpp

```diff
@@ -1048,8 +1048,8 @@ but the returned data may be incomplete.)");
 #endif
   options->addOption(
       SEARCH_THREADS_LIMIT,
-      "Max number of threads that could be used to process ArangoSearch "
-      "indexes during SEARCH operation",
+      "The maximum number of threads that can be used to process ArangoSearch "
+      "indexes during a SEARCH operation of a query",
```

Comment:
nit: according to the conventions you introduced a while ago for all startup options descriptions, they should all end with a full stop.
```suggestion
      "indexes during a SEARCH operation of a query.",
```